google/ggrc-core
# GGRC-4842 Remove Issue from /related_objects endpoint results.

# Issue description

* `Issue` isn't used from result provided by /related_object endpoint.
* Remove this functionality.

# Steps to test the changes

Open console in browser. Press `Edit Assessment` button in menu and check response for /related_objects request.

# Solution description

* Change `related_objects` method in assessment resourse.
* Add test for check this functionality.

# Sanity checklist




# PR Review checklist
